Makefile

```diff
@@ -15,4 +25,23 @@ update-charm-libs:
 check-dashboard-updates:
 	./scripts/check-dashboard-updates.sh
 
-.PHONY: help update-charm-libs check-dashboard-updates
+clean:
+	@echo "clean files"
+	@git clean -ffXd -e '!.idea' -e '!.vscode'
+	@echo "Cleaning existing build"
+	@rm -rf ${PROJECTPATH}/${CHARM_NAME}*.charm
+	@charmcraft clean
+	@echo "Remove download snap"
+	@rm -rf ${PROJECTPATH}/*.snap
```

Comment:
I think we should follow same approach as we are doing with charms, e.g. removing `${PROJECTPATH}/golang-openstack-exporter*.snap` instead of any snap.

Reply:
This project should not include any snap, and the snap can be download from git release assert easily.

Reply:
It's up to me, what I put in the repo directory :D, so that's why I think it would be great not remove any of my files.

Reply:
Ok, let me update it.